There appears to be another overflow bug:

````
--- FAIL: TestRuntime (0.39s)
    --- FAIL: TestRuntime/lua/basic.quotas.lua (0.04s)
        runtests.go:77: [output line 5, source line 44] expected: "done\t3736710778780434371", got: "done\t-980107325"
FAIL
````

I'll try to expand this pull request accordingly.

--- a/runtime/value.go
+++ b/runtime/value.go
@@ -303,7 +303,7 @@ func (v Value) ToString() (string, bool) {
 	}
 	switch x := v.iface.(type) {
 	case int64:
-		return strconv.Itoa(int(v.AsInt())), true
+		return strconv.FormatInt(v.AsInt(), 10), true
 	case float64:
 		return strconv.FormatFloat(v.AsFloat(), 'g', -1, 64), true
 	case bool:

